flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/common/TieredStorageUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.partition.hybrid.tiered.common;
+
+import org.apache.flink.runtime.io.network.partition.hybrid.tiered.storage.TieredStorageMemoryManager;
+
+/** Utils for reading or writing to tiered store. */
+public class TieredStorageUtils {
+    public static boolean needFlushCacheBuffers(
+            TieredStorageMemoryManager storageMemoryManager, float numBuffersTriggerFlushRatio) {
+        int numTotal = storageMemoryManager.numTotalBuffers();
+        int numRequested = storageMemoryManager.numRequestedBuffers();
```

Comment:
These are non-atomic now. Why not add a `getUtilization` for the memory manager.

Reply:
What if the requested buffers are not completed yet?

Reply:
We have removed the two Apis of `numTotalBuffers ` and `numRequestedBuffers`  because we have merged `TieredStorageMemoryManager` and `CacheFlushManager`. So we do not need expose the two args any more.